src/components/Label.jsx

```diff
@@ -0,0 +1,15 @@
+const Themes = [
+  { id: 'theme-choice-one', value: '1' },
+  { id: 'theme-choice-two', value: '2' },
+  { id: 'theme-choice-three', value: '3' },
+];
```

Comment:
This has nothing to do with this component, let's move it to where it's needed and used.